decoimpact/data/parsers/parser_filter_extremes_rule.py

```diff
@@ -44,12 +45,21 @@ def parse_dict(self, dictionary: Dict[str, Any], logger: ILogger) -> IRuleData:
         ]
         output_variable_name: str = get_dict_element("output_variable", dictionary)
         description: str = get_dict_element("description", dictionary, False) or ""
-        extreme_type: str = get_dict_element("extreme_type", dictionary)
-        self._validate_extreme_type(extreme_type)
+
+        extreme_type_name = "extreme_type"
+        extreme_type: str = get_dict_element(extreme_type_name, dictionary)
```

Comment:
can't we check if it's a string without having to introduce a new function and a second variable (extreme_type_name)

Reply:
This was in preperation for other rules to reuse this. And otherwise I'm copying the validation error each time (using this function 3 times in here). And I made an extra variable because I need the string in both the get_dict_element and the validate_extreme_type function, so I don't type the variable name twice

Reply:
allright :)